packages/app/src/search/actions/search.action.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+import { createAsyncThunk } from "@reduxjs/toolkit"
+import { SearchEvent } from "App/search/constants"
+import { SearchParams, SearchResult } from "App/search/dto"
+import { searchRequest } from "App/search/requests"
+
+export const search = createAsyncThunk<SearchResult | undefined, SearchParams>(
+  SearchEvent.Search,
+  async (payload) => {
+    const result = await searchRequest(payload)
+    // AUTO DISABLED - fix me if you like :)
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-return
```

Comment:
This warning seems like could be easy to handle 😉 

Decision team
1. Delete `no-unsafe-return` rule from eslint config, or
1. Handle warning as part of this PR 😉

Reply:
I would vote for leaving `no-unsafe-return` in eslint config. At least for me, it is a useful rule.